### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by Kadaaran**
As requested by @mozzadrella we have a pretty awesome `README.md` with all the technicals informations but having documentation on **How to use Classroom** would be really nice ✨ 

This is the first file originally from [the blog](https://blog.github.com/2018-03-06-How-to-use-group-assignments-in-GitHub-Classroom/), we can add more in the future 🙌 

Ideas:
- How to create a Classroom
- How to use the students roster

### CODE REVIEW

**Comment by Kadaaran** [comment]
@d12 It was to separate the code content from documents but I think the convention is without a `.` I'll change it 👍

**Comment by mozzadrella** [comment]
@Kadaaran friendly bump on this pls :)

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ With Classroom, you can spin up your course on GitHub and move on to the good st
 
 *Works with your Learning Management System (LMS).* Students submit a link to their assignment repository to your learning management system. Give feedback through comments in GitHub, but keep grades in your LMS.
 
-*You’re not alone.* The experts on the GitHub Education team are here to answer any of your questions, and we’ve got docs, best practices, and a strong community of educators to help you migrate to Classroom.
+*You’re not alone.* The experts on the GitHub Education team are here to answer any of your questions, and we’ve got [docs](https://github.com/education/classroom/tree/master/.docs), best practices, and a strong community of educators to help you migrate to Classroom.
```
Comment (johndbritton) [inline README.md]: @Kadaaran this is a broken link, should go to `/docs` not `/.docs`

Comment (Kadaaran) [reply README.md]: Fixed 👍

**Pull request merged**